src/clj/commiteth/routes/services.clj

```diff
@@ -60,21 +60,41 @@
                                    (map update-enabled repos)})
                github-repos))))
 
+(def bounty-renames
+  ;; TODO this needs to go away ASAP we need to be super consistent
```

Comment:
Agreed on this for sure! Feels like it could warrant its own issue/bounty. Were you envisioning that this should be the job of a different layer, closer to where we deal with posgres directly?

Reply:
Now that we're on clojure 1.9 I would vote for the second option. I think it would be helpful to point new and old contributors alike to that spec to get an idea of the domain model too. Just drafted up an issue for that here: https://github.com/status-im/open-bounty/issues/425